src/util/haptics.ts

```diff
@@ -0,0 +1,60 @@
+import { Capacitor } from '@capacitor/core'
+import { Haptics, ImpactStyle, NotificationType } from '@capacitor/haptics'
+
+/** True if the platform supports haptic feedback. */
+const hapticsSupported = Capacitor.isNativePlatform() && Capacitor.isPluginAvailable('Haptics')
```

Comment:
As far as I can tell, `hapticsSupported` is superfluous. Haptics.impact appears to be an innocuous NOOP on desktop works It doesn't produce any warnings or errors either. Do you see any reason to include this check?

If we do remove the condition, we should still keep the module since it is helpful for debugging, insulates our codebase from Haptics plugin API changes, and provides a hook for sound effects if they are added in the future.

Reply:
I'll do a quick test myself to make sure it's safe, and then plan to remove `hapticsSupported`. That was in the original PR, so I assumed it was necessary without verifying.

Reply:
Removed: 208c0a1e1a7001472ca17d4b67f544b1c7424205